test/cpp/qps/driver.cc

```diff
@@ -623,6 +632,24 @@ std::unique_ptr<ScenarioResult> RunScenario(
   // the result.
   bool client_finish_first =
       (client_config.rpc_type() != STREAMING_FROM_SERVER);
+  
+  if (!prometheus_query_time_file.empty()) {
+    std::ofstream timestamps_outfile;
+    timestamps_outfile.open(prometheus_query_time_file);
```

Comment:
Nit: You can supply the filename in the constructor for the `std::ofstream` object.